Makefile

```diff
@@ -20,6 +20,8 @@ ifeq ($(INTERACTIVE), 1)
 	DOCKER_FLAGS += -t
 endif
 DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
+# Use =, not :=, so that this is not expanded when we are not running unit tests.
+UNIT_TEST_PACKAGES = $(shell find -name '*_test.go' | sed 's,/[^/]*$$,,' | sort -u | grep -v '^\./\(integration\|vendor\)$$')
```

Comment:
> go list ./... does not automatically filter integration and vendor subdirectories; are you saying that should somehow happen?

`go list ./... | grep -v ^vendor/|integration/` (or something like this)

> go list lists all packages; this only finds packages which contain tests. I am unsure about which is better; go list causes ? $package_name [no test files] lines to be output, arguably both useful and useless. Right now the find -name '*.test.go' variant means $(UNIT_TEST_PACKAGES) is empty and make test-unit does not even start the container, but this difference will go away with the first added unit test.

I'm ok with the "no tests found" line (cause somehow helps figuring instantly when running tests  coverage)